src/styles/blocks.scss/stars.scss

```diff
@@ -0,0 +1,17 @@
+.stars__star {
+  background-image: url(../images/star.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+  height: 16px;
+  width: 16px;
+  margin: 0 2px;
+}
+
+.stars--4 > .stars__star:nth-child(-n+4) {
+  background-image: url(../images/star-active.svg);
+}
```

Comment:
implement stars for all modifiers, for 1,2,3,4,5 active stars(copy from `stars` task)